src/sentry/workflow_engine/endpoints/organization_detector_workflow_index.py

```diff
@@ -0,0 +1,140 @@
+from django.db import IntegrityError
+from drf_spectacular.utils import extend_schema
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import OrganizationEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_CONFLICT,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.parameters import GlobalParams
+from sentry.workflow_engine.endpoints.serializers import DetectorWorkflowSerializer
+from sentry.workflow_engine.endpoints.validators.detector_workflow import DetectorWorkflowValidator
+from sentry.workflow_engine.models.detector import Detector
+from sentry.workflow_engine.models.detector_workflow import DetectorWorkflow
+from sentry.workflow_engine.models.workflow import Workflow
+
+
+@region_silo_endpoint
+class OrganizationDetectorWorkflowIndexEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+        "DELETE": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ISSUES
```

Comment:
realizing we probably need to add `permission_classes` here, but I'm not sure which permissions to use?